eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -387,6 +387,8 @@ jobs:
     name: azsdk-pool-mms-win-2019-general
     vmImage: MMS2019
   steps:
+    - template: /eng/common/pipelines/templates/steps/check-spelling.yml
+
```

Comment:
question: is this new line necessary in yml (and has some semantic meaning), or is it just cosmetic?